ibm_catalog.json

```diff
@@ -365,7 +380,7 @@
           ]
```

Comment:
move `"key": "cbr_rules"` after `existing_rabbitmq_instance_crn`. following the rule that inputs that are not in redis, should be placed at the end

<img width="1298" alt="image" src="https://github.com/user-attachments/assets/aa866020-ef84-431b-8624-e0ab53735966" />

Reply:
check security-enforced as well